frontend/test/metabase/scenarios/collections/collections.cy.spec.js

```diff
@@ -529,6 +529,10 @@ describe("scenarios > collection_defaults", () => {
         .find(".Icon-chevronright")
         .should("not.exist");
     });
+
+    it.only("dasda", () => {
```

Comment:
don't forget this guy

Reply:
Hahahah, great catch! Sorry about that 😊